src/m365/teams/commands/conversationmember/conversationmember-list.ts

```diff
@@ -0,0 +1,186 @@
+        return this.getChannelId(teamId, args);
+      }).then((channelId: string) => {
+        let endpoint: string = `${this.resource}/v1.0/teams/${this.teamId}/channels/${channelId}/members`;
+        return this.getAllItems(endpoint, logger, true);
+      }).then((): void => {
+        if (args.options.output === 'json') {
+          logger.log(this.items);
+        } else {
+          logger.log(this.items.map((c: ConversationMember) => {
+            return {
+              id: c.id,
+              displayName: c.displayName,
+              userId: c.userId,
+              email: c.email
+            }
+          }));
+        }
+
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --teamId [teamId]',
+        description: 'The ID of the team where the channel is located. Specify either teamId or teamName, but not both.'
+      },
+      {
+        option: '--teamName [teamName]',
+        description: 'The name of the team where the channel is located. Specify either teamId or teamName, but not both.'
+      },
+      {
+        option: '-c, --channelId [channelId]',
+        description: 'The ID of the channel for which to list members. Specify either channelId or channelName, but not both.'
+      },
+      {
+        option: '--channelName [channelName]',
+        description: 'The name of the channel for which to list members. Specify either channelId or channelName, but not both.'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(args: CommandArgs): boolean | string {
+    if (args.options.teamId && args.options.teamName) {
+      return 'Specify either teamId or teamName, but not both.';
+    }
+
+    if (!args.options.teamId && !args.options.teamName) {
+      return 'Specify teamId or teamName, one is required';
+    }
+
+    if (args.options.teamId && !Utils.isValidGuid(args.options.teamId as string)) {
+      return `${args.options.teamId} is not a valid GUID`;
+    }
+
+    if (args.options.channelId && args.options.channelName) {
+      return 'Specify either channelId or channelName, but not both.';
+    }
+
+    if (!args.options.channelId && !args.options.channelName) {
+      return 'Specify channelId or channelName, one is required';
+    }
+
+    if (args.options.channelId && !Utils.isValidTeamsChannelId(args.options.channelId as string)) {
+      return `${args.options.channelId} is not a valid Teams ChannelId`;
+    }
+
+    return true;
+  }
+
+  private getTeamId(args: CommandArgs): Promise<string> {
+    if (args.options.teamId) {
+      return Promise.resolve(args.options.teamId);
+    }
+    
+    return new Promise<string>((resolve: (channelId: string) => void, reject: (error: any) => void): void => {
+      const teamRequestOptions: any = {
```

Comment:
Please use the group endpoint. See comments under #1918

Reply:
Now fixed and PR was updated!